core/embed/rust/storage.h

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
It does not need :) Deleted in [39ad89d](https://github.com/trezor/trezor-firmware/pull/2214/commits/39ad89d8ff73f886b38ddfcc03ba4ff4527d3674)

Reply:
> please also investigate why we need the cache song-and-dance for `get_experimental_features`.
> 
> My first suspicion is that it is because we want the field to be public but it is private.

I have found the connected [PR](https://github.com/trezor/trezor-firmware/pull/1565) and [commit](https://github.com/trezor/trezor-firmware/commit/96fd347ca895b6fc57bec7f3746fd791648d0931) and it seems to solve some issue that was in Trezor before that - something with workflow restarts